Filtered_kernel/include/CGAL/Filtered_predicate.h

```diff
@@ -59,6 +59,7 @@ class Filtered_predicate
   // AP::result_type must be convertible to EP::result_type.
 
   Filtered_predicate()
+    : c2e{}, c2a{}
```

Comment:
Does it even complain about not initializing trivial empty types? :roll_eyes: 
I think this patch is ok, although I would have used parentheses for consistency with the other constructors. Don't you need to do the explicit constructor below as well?

Reply:
My understanding is that trivial empty types (AKA class types) are always default initialised, so this does look like Coverity made a false positive, however, C2E, and C2A are template parameters, so I presume that the tool thinks you can pass pointer or scalar types for those parameters.  
e.g:
```c++
Filtered_predicate<EP,AP,C2E*,C2A*> pred;
```
That was my assumption.